fe/fe-core/src/main/java/com/starrocks/common/proc/StatisticProcDir.java

```diff
@@ -129,14 +129,20 @@ public ProcResult fetchResult() throws AnalysisException {
                     OlapTable olapTable = (OlapTable) table;
 
                     for (Partition partition : olapTable.getAllPartitions()) {
+                        boolean useStarOS = partition.isUseStarOS();
                         short replicationNum = olapTable.getPartitionInfo().getReplicationNum(partition.getId());
                         ++dbPartitionNum;
                         for (MaterializedIndex materializedIndex : partition
                                 .getMaterializedIndices(IndexExtState.VISIBLE)) {
                             ++dbIndexNum;
                             for (Tablet tablet : materializedIndex.getTablets()) {
-                                LocalTablet localTablet = (LocalTablet) tablet;
                                 ++dbTabletNum;
+
+                                if (useStarOS) {
```

Comment:
dbReplicaNum ignores StarOS Tablet, is this right?

Reply:
Yes, i think this refers to the storage replicas of tablet, but StarOS tablet only has computing replicas.
We may add more columns to distinguish between storage replicas and computing replicas later.